packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -46,12 +49,15 @@ class Authorized extends React.Component<Props> {
   }
 }
 
+type InjectedProps = {
+  [propName: string]: boolean;
+};
```

Comment:
This part I never managed to type "100%" in flopflip. You pass a `propName` of type string. Whatever is passed there would be part of the type here again. As such now the type is quite vague.